core/metrics-core-api/src/main/java/org/hawkular/metrics/core/api/MetricsService.java

```diff
@@ -144,6 +144,7 @@
      * @return An {@link Observable} that emits the results with the same ordering as funcs
      * @see Aggregate
      */
+    @SuppressWarnings("unchecked")
```

Comment:
SafeVarArgs is allowed on static and final methods. So you can make the 
method final and annotate it.